### DISCUSSION TITLE
Problems due to moving from zwavejs2mqtt to zwave-js-ui

### DISCUSSION THREAD

**Discussion by Sarcas666** [zwave-js/zwave-js-ui#2697] [Q&A]
I must have missed a warning somewhere... I am using zwavejs2mqtt together with Domoticz for quite a while now, without any real problems. I was running zwavejs2mqtt  7.something and saw that there was a new version. A glance at the README.md did not show any warnings, and neither did a glance at ISSUES. 

Updating went fine. But unfortunately, zwave-js-ui created new unique identifiers for all my devices and now my devices are doubles in Domoticz. I haven't figured out how to repair this, since I am still in the process of banging my head to the wall whilst tearing out my hair and wailing loudly.

Any advice is welcome...

**Comment by robertsLando** [comment]
@Sarcas666 I'm sorry for that. The problem is due to [this](https://github.com/zwave-js/zwave-js-ui/pull/2650/files#diff-13d707676a316f80450af7e926559a262caf619230db6372e8fcb3f801d29d97L852) line change. I will revert this asap so you will have it back to work like before (just need to delete duplicated entities)

**Reply by robertsLando** [reply]
https://github.com/zwave-js/zwave-js-ui/releases/tag/v8.0.2 is out now

**Reply by Sarcas666** [reply]
Super! I see it'll take roughly an hour before docker is updated. I'll try it asap :D
Thanks!
